spec/requests/api/v1/job_applications/job_application_show_spec.rb

```diff
@@ -109,6 +109,18 @@
         expect(json[:message]).to eq("Job application not found")
         expect(json[:status]).to eq(404)
       end
+
+      it "returns error serializer if no params are passed in URL for job application id" do
+        get "/api/v1/users/#{@user.id}/job_applications/"
+
+        expect(response).to_not be_successful
+        expect(response.status).to eq(400)
+
+        json = JSON.parse(response.body, symbolize_names: true)
+
+        expect(json[:message]).to eq("Job application ID is missing")
+        expect(json[:status]).to eq(400)
+      end
```

Comment:
Thank you for adding this test and being very through with your work. I like your error message here, as I feel it is very informative and user-friendly.